cmd/ctrlc/root/run/exec/runner.go

```diff
@@ -67,13 +128,34 @@ func (r *ExecRunner) Start(job api.Job) (string, error) {
 		return "", fmt.Errorf("failed to unmarshal job agent config: %w", err)
 	}
 
+	client, err := api.NewAPIKeyClientWithResponses(
+		viper.GetString("url"),
+		viper.GetString("api-key"),
+	)
```

Comment:
you are making it so the job agent has to be run inside a command. This should be decoupled.